app/services/pledge_form_generator.rb

```diff
@@ -92,34 +93,34 @@ def generate_pledge_pdf
   def build_header(pdf)
     y_position = pdf.cursor
     pdf.bounding_box([0, y_position], width: 200, height: 100) do
-      pdf.image "#{Rails.root.join("public", "#{FUND.downcase}_logo.png")}",
+      pdf.image "#{Rails.root.join("public", "#{@fund.name.downcase}_logo.png")}",
                 position: :left,
-                width: CONFIG_DATA[FUND][:width],
-                height: CONFIG_DATA[FUND][:height]
+                width: CONFIG_DATA[@fund.name][:width],
+                height: CONFIG_DATA[@fund.name][:height]
```

Comment:
okay, here's where this wound up:
* add a config setting to the Fund object
* adjust pledge generator + the view to use that setting
* change the seeds so that pledge generation is turned off by default

You can try it out by running `Fund.update_all pledge_generation_config: 'DCAF'`. It'll be a little weird since some of the config is in the PDF class and some of it is in tenant, but it'll come out clean in production.

I'll also note that pledge generation is kind of overdue for an overhaul - I think this strikes a balance between 'getting this working now' and also 'still keeps things open for further development'.